src/bayes_optuna/optuna_hpo.py

```diff
@@ -112,7 +112,7 @@ def perform_experiment(self):
             trial_result = self.trialDetails.trial_result
         finally:
             self.resultsAvailableCond.release()
-        trial_result = "prune"
+        trial_result = "failure"
```

Comment:
Wondering, will this not cause us to always return failure?

Reply:
removed it now as it was added by mistake